minidec/main.rs

```diff
@@ -59,6 +67,7 @@ fn main() {
         //println!("Local Variable info: {:?}", rfn.locals);
         let mut writer: IRWriter = Default::default();
         writer.emit_il(Some(rfn.name.clone()), &rfn.ssa, &mut ff);
+        writer.emit_il(Some(rfn.name.clone()), &rfn.ssa, &mut ffm);
```

Comment:
This works for the String, but has problems with the file.
`error: the trait bound 'std::fs::File: std::fmt::Write' is not satisfied [E0277]`
I will write the String to the respective files. Will implement this when I have time.